example/src/App.js

```diff
@@ -12,9 +12,11 @@ import image3 from './wallpapers/Mountain-Range.jpg';
 import image4 from './wallpapers/Pink-Forest.jpg';
 import image5 from './wallpapers/Snow.jpg';
 import image6 from './wallpapers/Lion.jpg';
-import { SimpleImg, SimpleImgProvider } from 'react-simple-img';
+import { SimpleImg, init  } from './src/index';
 import './App.css';
 
+init();
```

Comment:
I think its better to call `init()` in `index.js` since it's a once-off global step.